### ISSUE TITLE
Store probing area settings on EEPROM(cnc use)

### ISSUE THREAD

**Issue by yOPERO** [MarlinFirmware/Marlin#926]
Hi all

I am currently using marlin with a small CNC dedicated to mill PCBs, I found that the Zprobe auto-levelling feature is working nicely and when it comes to PCB milling this is a "must" have feature.

Normally the sizes of the PCBs change a lot and so does the area to be probed, I would like to get a guidance on how to make possible to store the following parameters on the eeprom and then be able to pull that info from eeprom to marlin(same as steps /mm,etc)
 #define LEFT_PROBE_BED_POSITION 15
 #define RIGHT_PROBE_BED_POSITION 170
 #define BACK_PROBE_BED_POSITION 180
 #define FRONT_PROBE_BED_POSITION 20

To summarize my aim is to be be able to edit the probing area settings on the fly without re-uploading the firmware.

Regards,

**Comment by Roxy-3D** [comment]
###Hi Alex,
###
###The printrboard only compiles with the "3 points probe" there is not enough memory 
###for the other method.
###
###So would be a pity to lose it in the main marlin.

I did all of the development of the 'Enhanced G29' code on a PrintrBoard.   I assure you, I have lots of room for that and a bunch of other code!    My vote is to remove the '3 point probe' code.   It causes a lot of problems if the print extends out to the corner that is not probed!

**Comment by boelle** [comment]
is this one still relavant to have open?

**Comment by CONSULitAS** [comment]
@boelle see #1185 

I will need CNC features, too. Plan to mill small PCBs. Holder for Dremel mill printed yesterday. Seems to fit for my needs. We will see...

**Comment by boelle** [comment]
i'm closing this one... no one have shown active interest.... also it looks more like a hardware support issue which we have agreed on should come from people who need it or want it. ie if vendors want support then they shuould submit a PR
